### ISSUE TITLE
Can't use custom docker image?

### ISSUE THREAD

**Issue by legolord208** [japaric/cross#109]
I have this Cargo.toml:
```
[package]
name = "play-youtube"
version = "0.1.0"
authors = ["LEGOlord208 <LEGOlord208@krake.one>"]

[dependencies]
# dependencies here

[target.x86_64-unknown-linux-gnu]
image = "play-youtube"
```

I have the `play-youtube` docker image with the `latest` tag.

Using `cargo build --release -v --target x86_64-unknown-linux-gnu`,
I just see that it doesn't use the image - AND it prints
`warning: unused manifest key: target.x86_64-unknown-linux-gnu.image`

Help?

**Comment by Emilgardis** [comment]
Hello Legolord! Nice to see you trying to use cross.

So, the image tag is supposed to go into `Cross.toml`, not `Cargo.toml`.
https://github.com/japaric/cross#custom-docker-images

Also, you'll have to run `cross build --release --target x86_64-unknown-linux-gnu` so that you actually launch cross. Hopefully that fixes things.

**Comment by legolord208** [comment]
Ohh! That makes sense! Nice to see it's a separate config. Sorry for not reading carefully enough.

(Though, `cargo build` was a typo. I meant `cross build` there.)

**Comment by Alexhuszagh** [comment]
This was much more satisfactorily implemented in #678.
